pkgs/development/tools/twitch-cli/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "twitch-cli";
+  version = "1.1.19";
+
+  src = fetchFromGitHub {
+    owner = "twitchdev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-osR7+oQ0QBIk/OrMb4Txhb31zELi+eoM47GsWgF/PAg=";
+  };
+
+  vendorHash = "sha256-OhcRMXY8s+XciF+gV3cZ8fxtzo9+I76tBPZ0xG8ddHU=";
+
```

this fixes `twitch-cli version`

```suggestion

  ldflags = [
    "-s"
    "-w"
    "-X=main.buildVersion=${version}"
  ];

```